Source/WebKit/Shared/SessionState.h

```diff
@@ -125,12 +125,17 @@ class FrameState : public RefCounted<FrameState> {
     Vector<Ref<FrameState>> children;
 
 private:
+    FrameState(String&& urlString, String&& originalURLString, String&& referrer, AtomString&& target, std::optional<WebCore::FrameIdentifier>, std::optional<Vector<uint8_t>>&& stateObjectData, int64_t documentSequenceNumber, int64_t itemSequenceNumber, WebCore::IntPoint scrollPosition, bool shouldRestoreScrollPosition, float pageScaleFactor, std::optional<HTTPBody>&&, std::optional<WebCore::BackForwardItemIdentifier>, std::optional<WebCore::BackForwardFrameItemIdentifier>, bool hasCachedPage, String&& title, WebCore::ShouldOpenExternalURLsPolicy, RefPtr<WebCore::SerializedScriptValue>&& sessionStateObject, bool wasCreatedByJSWithoutUserInteraction, bool wasRestoredFromSession,  std::optional<WebCore::PolicyContainer>&&,
```

Comment:
Does this get called somewhere in this patch? Cmd-F isn't finding it. (If not called, probably best to leave out.)

Reply:
It gets used by the GeneratedSerializer IPC code, which initializes a FrameState using a bunch of Moves of the decoded members. I put a breakpoint in the old constructor (before adding this) and hit it in the serializer, so it seemed like we were doing a bunch of copies we don't need to.